qiita_db/sql_connection.py

```diff
@@ -402,9 +430,16 @@ def execute_fetchall(self, sql, sql_args=None):
         """
         with self._sql_executor(sql, sql_args) as pgcursor:
             result = pgcursor.fetchall()
-        return result
+            if return_types:
```

Comment:
Why not always return result, types and simply ignore the second return? Other option is to have execute_fetchall_with_types.